uswds/uswds
# USWDS - Header: Remove FOUC on extended header with mega-menu.

## Description

Closes #4047. Adds a more specific selector to extended header with mega-menu to remove the flash of unstyled content.

## Additional information
Steps to test:

1. Go to [Header compare page](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-extended-megamenu-flash/components/preview/header--default.html)
1. Open devtools
1. In firefox go to `Network → No Throttling ↕ → Regular 2G` 
  In Chrome  `Network → Online ▼ → Slow 3G`
1. Hard refresh page 

Before you hit Submit, make sure you’ve done whichever of these applies to you:
